coins.json

```diff
@@ -1065,5 +1065,22 @@
             "clientPublic": "",
             "clientDocs": "https://fio.foundation"
         }
+    },
+    {
+        "id": "harmony",
+        "name": "Harmony",
+        "symbol": "ONE",
+        "decimals": 18,
+        "blockchain": "Harmony",
+        "derivationPath": "m/44'/1023'/0'/0/0",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1Extended",
+        "explorer": "https://explorer.harmony.one/#/tx/",
+        "info": {
+            "url": "https://harmony.one",
+            "client": "https://github.com/harmony-one/go-sdk",
+            "clientPublic": "https://github.com/harmony-one/go-sdk",
```

Comment:
do we have public node for RPC?

Reply:
@rlan35 Can probably answer

Reply:
I checked, No public node yet for RPC

Reply:
we still need this when integrating into Trust Wallet, but you can provide later